applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,533 @@
+# Liberland Blockchain - the Judiciary
+
+- **Team Name:** Liberland Blockchain Team
+- **Payment Address:** Polkadot 13uikMrP8PvgapAhskyHgM6E8Yw7E6C9LxXMZBpgp2Dm8Nsi
```

Comment:
Please indicate what currency you would like to be paid in and whether you want the stablecoin and DOT payments to the same address (one will be on AssetHub, the other on Polkadot).

Reply:
pure DOT is fine